threedi_modelchecker/config.py

```diff
@@ -579,6 +579,25 @@ def scalar_subquery(query):
         message="v2_manhole.drain_level cannot be null when using sub-basins (v2_global_settings.manhole_storage_area > 0) and no DEM is supplied.",
     ),
 ]
+CHECKS += [
+    QueryCheck(
+        level=CheckLevel.WARNING,
+        error_code=108,
+        column=table.crest_level,
+        invalid=Query(table)
+        .join(
+            models.ConnectionNode,
+            table.connection_node_start_id == models.ConnectionNode.id
+            or table.connection_node_end_id == models.ConnectionNode.id,
```

Comment:
The `or` will not work, instead you can you can import `or_` from sqlalchemy, or use the bitwise or `|`.

See https://docs.sqlalchemy.org/en/20/core/operators.html#using-conjunctions-and-negations

Reply:
fixed